whisper/message_test.go

```diff
@@ -351,7 +351,7 @@ func TestRlpEncode(t *testing.T) {
 	}
 
 	var decoded Envelope
-	rlp.DecodeBytes(raw, &decoded)
+	rlp.DecodeBytes(raw, &decoded) // nolint: errcheck
```

Comment:
We can disable `errcheck` for `./whisper`. Check out https://github.com/status-im/status-go/blob/develop/.golangci.yml#L76-L79.

It will be something like:
```
- path: whisper/.*_test.go
  linters:
    - errcheck
    - gosec
```